qcfractal/queue/managers.py

```diff
@@ -428,9 +460,26 @@ def update(self, new_tasks: bool = True, allow_shutdown=True) -> bool:
         self._update_stale_jobs(allow_shutdown=allow_shutdown)
 
         results = self.queue_adapter.acquire_complete()
+
+        # Stats fetching for running tasks, as close to the time we got the jobs as we can
+        last_time = self.statistics.last_update_time
+        now = self.statistics.last_update_time = time.time()
+        time_delta_seconds = now - last_time
+        try:
+            running_tasks = self.queue_adapter.count_running()
```

Comment:
Instead of running tasks why not count the number of running workers? The number of running workers is often available see `connected_workers` [here](https://parsl.readthedocs.io/en/stable/stubs/parsl.executors.HighThroughputExecutor.html) and [Dask](https://github.com/dask/dask-jobqueue/blob/bff96a329439c23414bd0cb11c0ee8e67145d96b/dask_jobqueue/core.py#L485).

Reply:
Because counting the running tasks is a better gauge of how many resources are being consumed. If we only count workers, then we don't take into account that each worker can have multiple tasks running on it. I did try to work it out such that cores cannot be oversubscribed, but I don't know how effective that is.